### PR TITLE
Update to 'new'-style string formatting

### PR DESCRIPTION
**Pull request by Roel**
* [x] I have read and followed the guidelines in the `CONTRIBUTING` document
* [x] I have checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.
* [x] All new and existing tests passed.

Update to string formatting using the .format() function.
There should be no impact on existing implementations or use-cases.

### CODE REVIEW

File: pydov/types/abstract.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def _parse(cls, func, xpath, namespace, returntype):
 
         """
         if namespace is not None:
-            ns = '{%s}' % namespace
+            ns = '{{{}}}'.format(namespace)
```
Comment (pjhaest) [inline pydov/types/abstract.py]: New to me, why the triple accolades?

**Comment by pjhaest** [comment]
@Roel : do you have more questions? If not, merging is ok I guess.

File: pydov/types/abstract.py (hunk 1 above)
Comment (Roel) [reply pydov/types/abstract.py]: To get a literal `{` you have to escape it by doubling. The middle pair is substituted by `namespace`. Looks a bit odd, but this sequence of `{{{}}}` is to get `{namespace}`. See also https://docs.python.org/3.7/library/string.html#format-string-syntax

Comment (stijnvanhoey) [reply pydov/types/abstract.py]: so, if updating to new string formatting, why not 

```
ns = f'{{namespace}}'
```

Comment (Roel) [reply pydov/types/abstract.py]: Because that syntax is only supported as of Python 3.6.

Comment (pjhaest) [reply pydov/types/abstract.py]: Ok, so I guess it's about time that I drop my request for Python 2.7 support :-)

**Pull request merged**